ext/build_defaults.yaml

```diff
@@ -131,8 +131,8 @@ osx_signing_cert: "Developer ID Installer: PUPPET LABS, INC. (VKGLGN2B6Y)"
 osx_signing_keychain: "/Users/jenkins/Library/Keychains/signing.keychain"
 osx_signing_server: 'osx-signer.delivery.puppetlabs.net'
 vanagon_project: TRUE
-apt_repo_name: 'PC1'
-yum_repo_name: 'PC1'
+apt_repo_name: 'puppet5'
+yum_repo_name: 'puppet5'
```

Comment:
you shouldn't need these at all anymore, since we just use `repo_name` set below.